pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -80,6 +80,61 @@ func NewNodeController(
 	}
 }
 
+// Generates num pod CIDRs that could be assigned to nodes.
+func generateCIDRs(num int) util.StringSet {
+	res := util.NewStringSet()
+	for i := 0; i < num; i++ {
+		// TODO: Make the CIDRs configurable.
+		res.Insert(fmt.Sprintf("10.244.%v.0/24", i))
```

Comment:
I have a diff sitting in my tree to make this configurable via env variable (something like KUBE_CLASS_B). Can you fix this sooner rather than later? It affects things like the ability to spin up multiple clusters in the same network, and networks are a slightly harder resource to come by.

(Yes, I realize it means some ugly salt config. :( )